.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,100 @@
+name: Publish NuGet
```

Comment:
The project uses AppVeyor as a build server.
Let's continue to use it.

Reply:
You can leave the build stuff to me.

Reply:
I will leave the build stuff to you. Thanks.